github_connector_odoo/monkeypatching.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import odoo.release as release
+from odoo.modules import module
+
+adapt_version_original = module.adapt_version
+
+
+def adapt_version(version):
+    """Overwrite the method to avoid the error if use values that do not contain serie
+    (17.0), for example 13.0.4.0.0.0
```

Comment:
I think the example should `4.0`, not a complete version containing even more 0s.